golem-cli/src/command/component.rs

```diff
@@ -59,6 +63,10 @@ pub enum ComponentSubCommand<ProjectRef: clap::Args, ComponentRef: clap::Args> {
         /// Update mode - auto or manual
         #[arg(long, default_value = "auto", requires = "try_update_workers")]
         update_mode: WorkerUpdateMode,
+
+        /// Do not ask for creating a new component in case it does not exists
+        #[arg(short = 'y', long)]
```

Comment:
why is it 'y'? looks like as if it would imply saying yes to the interactive question, but it seems to be like the opposite

Reply:
I don't like short parameters :D Copied it from the other places where the same flag existed already (added yesterday)
What would you use?

Reply:
i would either change it to behave like "yes", or change it to --no? not sure :D